core/templates/dev/head/filters.js

```diff
@@ -407,6 +409,21 @@ oppia.filter('stripFormatting', [function() {
   };
 }]);
 
+oppia.filter('getAbbreviatedText', [function () {
+  return function (text, characterCount) {
```

Comment:
Is it possible to reference the limit directly here? It would reduce code complexity.

Reply:
@kevinlee12 If in future it is to be used somewhere else then it would be easier to specify character limit of that feature and use this function.